doc/source/Getting_started/Troubleshooting.rst

```diff
@@ -0,0 +1,168 @@
+     - GRPC default
+       PythonNET: pyaedt.settings.use_grpc_api = False
+
+Above options are available only in windows. Linux uses GRPC in any version as it is the only way to connect
+to AEDT API.
+
+
+How to check that Electronics Desktop API is working and correctly configured
+-----------------------------------------------------------------------------
+To start Electronics desktop server in gRPC mode use the following syntax:
+
+.. code:: python
+
+   path\to\AnsysEM\v231\Win64\ansysedt.exe -grpcsrv 50001   # Windows
+   path\to\AnsysEM\v231\Lin64\ansysedt -grpcsrv 50352       # Linux
+
+The number on which the server is running is the port that AEDT should use to listen and receive
+any command from PyAEDT. This allows to have multiple AEDT sessions running on the same machine
+and listening on the same port.
+
+Check that AEDT GRPC API can run
+--------------------------------
+Native Electronics Desktop API command can be used to launch Electronics Desktop from command line.
+This can be done even without PyAEDT to check that everything is setup correctly and all environment
+variables have been correctly setup.
+
+.. code:: python
+
+    import sys
+    sys.path.append(r"ANSYSEM_ROOT231\PythonFiles\DesktopPlugin")
+    import ScriptEnv
+    print(dir())
+    ScriptEnv.Initialize("", False, "", 50051)
+    print(dir())
+
+
+
+Failures in connecting to GRPC API
+----------------------------------
+On Linux, it may happens that PyAEDT fails to initialize a new session of Electronics Desktop
+or to connect to an existing one.
+This may be due to:
+ - Firewall
+ - Proxy
+ - Permissions
+ - License
+ - Scheduler used to launch AEDT like LSF
+
+In case of issues with proxy, you may try the following environment variable:
+
+.. code:: python
+
+    export no_proxy=localhost,127.0.0.1
+
+Run your PyAEDT script. If it still fails, then try:
+
+.. code:: python
+
+    export http_proxy=
+
+Run your PyAEDT script. If the errors still persists, try the following:
+
+1. Check that AEDT starts correctly from command line using GRPC port option
+2. enable all debug log variables and check logs.
+
+.. code:: python
+
+    export ANSOFT_DEBUG_LOG=/tmp/testlogs/logs/lg
+    export ANSOFT_DEBUG_LOG_SEPARATE=1
+    export ANSOFT_DEBUG_LOG_TIMESTAMP=1
+    export ANSOFT_DEBUG_LOG_THREAD_ID=1
+    export ANSOFT_DEBUG_MODE=3
+
+
+Turn on the GRPC trace on the server side too:
+
+.. code:: python
+
+    export GRPC_VERBOSITY=DEBUG
+    export GRPC_TRACE=all
+
+Then start ansysedt.exe as GRPC server.
+
+.. code:: python
+
+    ansysedt -grpcsrv 50051
+
+The GRPC trace is printed on the terminal console. Capture the output as the server.txt file.
+In another terminal:
+
+.. code:: python
+
+    export GRPC_VERBOSITY=DEBUG
+    export GRPC_TRACE=all
+
+Run the PyAEDT script(make sure it is trying to connect to the same port as the GRPC server).
+Capture the output as the client.txt file. Send all the logs generated to Ansys Support.
```

Comment:
```suggestion
Run the PyAEDT script, making sure that it is trying to connect to the same port as the gRPC server.
Capture the output as the ``client.txt`` file.
Send all the logs generated to Ansys Support.
```
These lines should all be steps. Use an email address in the last line instead of Ansys support--and do we really want this output to be sent to Ansys Support or to the PyAnsys Core team?

Reply:
in this case specifically it is Ansys Support because it is an issue of the native API and are logs to be sent to our developers